packages/editor/src/AppModel/AppModel.ts

```diff
@@ -80,6 +80,11 @@ export class AppModel implements IAppModel {
     }
   }
 
+  changeComponentMapId(oldId: ComponentId, newId: ComponentId) {
```

Comment:
Since this function is always called after `Component.changeId`, maybe we can move this logic to `Component.changeId`?

Reply:
May should put `ComponentModel.changeId` into `AppModel.changeComponentMapId`? Because `AppModel` is the controller of `ComponentModel`.

Reply:
No. `ChangeId` is more reasonable and natural to be a method of `ComponentModel` than of `AppModel`.

Reply:
OK.